app/views/layouts/application.html.erb

```diff
@@ -8,14 +8,14 @@
   </head>
 
   <body>
-    <%= render "layouts/navigation" %>
+    <%= render user_signed_in? ? 'layouts/navigation' : 'layouts/sign_navigation' %>
+    <div class="container-fluid">
+      <%= yield %>
+    </div>
     <div id="flash">
       <% flash.each do |key, value| %>
```

Comment:
flash messages are expected to be under navigation not at the bottom

Reply:
solved